src/frontend/src/flows/recovery/recoveryWizard.json

```diff
@@ -0,0 +1,23 @@
+{
+  "en": {
+    "title": "Create your recovery phrase",
+    "paragraph": "This will take a few minutes. Make sure no one is watching your screen, and that you have a pen and paper or password manager ready. ",
```

Comment:
```suggestion
    "paragraph": "This will take a few minutes. Make sure no one is watching your screen and that you have a pen and paper or password manager ready.",
```

Reply:
Agree with @frederikrothenberger's updates